### DISCUSSION TITLE
ScriptReference/SceneManagement.Scene.IsValid

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#677] [Notes]
# ScriptReference/SceneManagement.Scene.IsValid

https://docs.unity3d.com/ScriptReference/SceneManagement.Scene.IsValid.html

**Comment by FreyaHolmer** [comment]
For new/untitled/unsaved scenes, `IsValid()` returns `true`. If you want to test if the scene doesn't have a corresponding **asset**, then instead check if `Scene.path` is an empty string
